File: back/src/main/java/com/tdevilleduc/urthehero/back/dao/PersonDao.java
package com.tdevilleduc.urthehero.back.dao;

import com.tdevilleduc.urthehero.back.model.Person;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;


@Repository
public interface PersonDao extends JpaRepository<Person, Integer> {

    Person findById(int id);
}

File: back/src/main/java/com/tdevilleduc/urthehero/back/dao/PersonDao.kt
package com.tdevilleduc.urthehero.back.dao

import com.tdevilleduc.urthehero.back.model.Person
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface PersonDao : JpaRepository<Person, Int> {
    fun findByLogin(userName: String): Optional<Person>
}